### PR TITLE
implement pkg handler remove test cases

### PR DESCRIPTION
**Pull request by kmrmt**
Signed-off-by: Kosuke Morimoto <ksk@vdaas.org>

### Description:

SSIA

### Environment:

- Go Version: 1.18.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [X] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vankichi** [comment]
@kmrmt 
the case is fine :)
please keep continuing implementation!

File: pkg/agent/core/ngt/handler/grpc/handler_test.go [hunk 1]
```diff
@@ -4618,90 +4618,356 @@ func Test_server_MultiUpsert(t *testing.T) {
...
-	}
-	type fields struct {
-		name              string
-		ip                string
-		ngt               service.NGT
-		eg                errgroup.Group
-		streamConcurrency int
+		ctx      context.Context
+		indexId  string
+		removeId string
```
Comment (vankichi) [inline pkg/agent/core/ngt/handler/grpc/handler_test.go:4628]:
```suggestion
		originId  string
		indexedId string
```
or
```suggestion
		indexId  string
		targetId string
```
umm... :thinking: 
Of course, I do not care if you won't change.

**Comment by kevindiu** [comment]
I think this changes needs to pass E2E, can you please enable it for this PR?

**Review by kevindiu — APPROVED** [review]

File: pkg/agent/core/ngt/handler/grpc/handler_test.go (hunk 1 above)
Comment (kmrmt) [reply pkg/agent/core/ngt/handler/grpc/handler_test.go:4628]: My intent is that `indexId` means the id which is in the Vald and `removeId` means  the id which is request for remove.
It is too difficult so let me put it in current expression for once.

**Review by vankichi — APPROVED** [review]

**Comment by vankichi** [comment]
@kpango CRE review has passed. Please review.

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**